webpack.config.js

```diff
@@ -0,0 +1,61 @@
+const path = require('path');
+const Dotenv = require('dotenv-webpack');
```

Comment:
@avpaul I see that you're not using es6 here, while importing the models, Is there a way we can also use es6 here ?

Reply:
https://github.com/andela/ah-kgl-avengers-frontend/pull/1#discussion_r289872727

Check the answer to @niyobobo  on the same question.

Reply:
Sorry for the git clone link in the description, it using the SSH format as it is the one I use. I copy pasted without changing it to the HTTPS format. But if you check the `package.json` file there is a correct link to the repo.

Reply:
I could also go on with the opinion that we make use of es6.

Reply:
@avpaul You did not address the third(3) point of the package-lock.json

Reply:
@aaronsekisambu  Now it's fixed

Reply:
@me-x-mi  What do we do?

Reply:
@avpaul I am trying to add a component that makes use of some of the es6 stuff for testing it. 
I would check so that we do not have issues in future.

Reply:
@aaronsekisambu and @avpaul I think we may need to have package-lock.json in our repo. 
In the past, I used to think there is no need or important. But, a senior developer told me that, its important is to lock the packages used. So if there lock file is there, if someone tries to change the version of a dependency, then we might probably easily realize. And also, it will help reduce risks of `it does not work in my side` as people will be using the same versions of dependencies. 
Any comments welcomed here.

Reply:
@me-x-mi  After reading [this question](https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5) I agree that we should add it.